layouts/partials/header.html

```diff
@@ -2,7 +2,7 @@
 
 <html lang="en">
   <head>
-    <title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ .Title }} - {{ .Site.Title }}{{ end }}</title>
+    <title>{{ if ! .IsHome }}{{ .Title }} - {{ end }}{{ .Site.Title }}</title>
```

Comment:
Can't find anything on that, but I do find that `if not` works so I'll change it :)